src/index.js

```diff
@@ -1 +1,24 @@
 import './style.css';
+import APIhelper from './APIHelper.js';
+
+const refreshBtn = document.getElementById('refresh-btn');
+const form = document.querySelector('form');
+const scoresList = document.querySelector('.scores__table ul');
+
+APIhelper.createGame();
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  APIhelper.postScore(form.name.value, form.score.value);
+  form.name.value = '';
+  form.score.value = '';
+  refreshBtn.click();
+});
+
+refreshBtn.addEventListener('click', async () => {
+  const data = await APIhelper.fetchScores();
+  if (data.result.length > 0) {
+    scoresList.classList.remove('d-none');
+  }
+  scoresList.innerHTML = data.result.map((obj) => `<li>${obj.user}: ${obj.score}</li>`).join('');
+});
```

Comment:
- [ ] Let's make sure that after the user reloads the page, the results from the leaderboard don't disappear. I can see that you have gone beyond the requirements, and the application creates a new game each time a user opens it. Unfortunately, the idea here is that the user can persist data through the API and access this even if he changes computers.

I will give you a couple of options here, since you have already worked on getting a new GameID, perhaps you can store it on LocalStorage and not fetch for a new game each time the user loads the page. You could also expand this, and show the id of the game to the user. He can then use it to access the same game through other computers with an input to submit ids.

Or just go with the classic approach, hardcode a GameId, where all of the users would participate in the same game.

Let me know if you have any doubts about this and I will leave a `gif` that references the unintended behavior:

![list-not-api](https://user-images.githubusercontent.com/79114233/146262812-4c4f160f-1d7b-4fea-9496-304d591c4be6.gif)